src/components/Calculator.js

```diff
@@ -1,42 +1,67 @@
 /* eslint-disable react/prefer-stateless-function */
 /* eslint-disable arrow-body-style */
+/* eslint-disable react/prop-types */
 import React from 'react';
 import '../Styles/style.css';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: '',
+      next: '',
+      operation: '',
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
- [ ] Please remove all linters disabled lines and fix linters issues one by one.